.github/workflows/docker_publish.yml

```diff
@@ -255,18 +264,21 @@ jobs:
           cd $GITHUB_WORKSPACE/tests
           ./travis-run-tests.sh python3
 
+  # if the previous step that tests the docker images passes then the images
+  # can be copied from the ghcr where they are saved using :ci_testing tags to
+  # :latest and :stable tags.
   pushing_test_stable_img:
     if: ${{ github.repository_owner == 'pyne' }}
     needs: [BuildTest]
     runs-on: ubuntu-latest
     strategy:
       matrix:
         pymoab : [
-          '',
+          # '', commented out as this combination is not built
           _pymoab,
           ]
         dagmc : [
-          '',
+          # '', commented out as this combination is not built
```

Comment:
I think I disagree with this.
I believe we did build 5 Img and tried to push 6.
the one this blank option would allow to build:
 - `no-pymoab -- no-dagmc`
 -  `pymoab -- no-dagmc`
which we should be building...
but the option `no-pymoab` & `dagmc` does not exist and should ignore in the matrix....

Reply:
see `ctr: failed to resolve reference "ghcr.io/pyne/ubuntu_18.04_py3_dagmc_pyne-deps:ci_testing`from https://github.com/pyne/pyne/runs/3690294186?check_suite_focus=true

Reply:
> > > This workflow only runs on push, so it won't run on the PR.
> > > I looked to your repo/branch, but it failed because image repos can't have an upper case name...
> > 
> > 
> > I've changed my username to be all lowercase now and have retriggered the action on my fork.
> > https://github.com/shimwell/pyne/actions/runs/1397725708
> > It looks like everything passed ✔️ but the last stage didn't run because of the `if github.repository_owner == 'pyne' ` part
> 
> You could change temporary it to `shimwell`

Thanks, Bam, I've changed to shimwell repo owner so I can test the whole workflow. I will change it back once everything is ready